src/main/java/appeng/container/implementations/ContainerCraftConfirm.java

```diff
@@ -87,7 +89,7 @@
 	public boolean noCPU = true;
 	@GuiSync( 7 )
 	public String myName = "";
-	protected long cpuIdx = Long.MIN_VALUE;
+	private final long cpuIdx = Long.MIN_VALUE;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "cpuIdx" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)